go.mod

```diff
@@ -52,18 +51,20 @@ require (
 	github.com/tonistiigi/fifo v0.0.0-20190816180239-bda0ff6ed73c // indirect
 	github.com/tonistiigi/fsutil v0.0.0-20191018213012-0f039a052ca1 // indirect
 	go.uber.org/zap v1.11.0 // indirect
-	golang.org/x/sys v0.0.0-20190924154521-2837fb4f24fe
+	golang.org/x/sys v0.0.0-20191127021746-63cb32ae39b2
 	k8s.io/api v0.0.0
 	k8s.io/apimachinery v0.0.0
 	k8s.io/client-go v0.0.0
 	k8s.io/component-base v0.0.0
 	k8s.io/klog v1.0.0
 	k8s.io/kubectl v0.0.0
-	k8s.io/kubernetes v0.0.0
+	k8s.io/kubernetes v1.13.0
```

Comment:
@nalind please revert this - we have a `replace` stanza where we set this to `v1.17.0` (using replace right now because the release candidate did not have a valid semver).

For what it's worth it appears that no kube bumps were changed.

Reply:
These are the values `update-deps.sh` changes them to.  Are you sure?

Reply:
if this is coming from `update-deps.sh`, let's keep it :/

Reply:
Okay, then.